.github/workflows/release.yaml

```
@@ -0,0 +1,175 @@
+        case ${{ matrix.job.target }} in
+          arm-unknown-linux-*) sudo apt-get -y update ; sudo apt-get -y install gcc-arm-linux-gnueabihf; sudo dpkg --add-architecture armhf; sudo apt-get update || true; sudo apt-get -y install libpoppler-glib-dev:armhf libpoppler-dev:armhf libcairo2-dev:armhf libglib2.0-dev:armhf; export PKG_CONFIG_LIBDIR=/usr/lib/arm-linux-gnueabihf/pkgconfig;;
+          aarch64-unknown-linux-gnu) sudo apt-get -y update ; sudo apt-get -y install gcc-aarch64-linux-gnu; sudo dpkg --add-architecture arm64; sudo apt-get update || true; sudo apt-get -y install libpoppler-glib-dev:arm64 libpoppler-dev:arm64 libcairo2-dev:arm64 libglib2.0-dev:arm64;;
+          aarch64-apple-darwin) NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"; brew install gnu-tar poppler; sudo ln -s $(brew --prefix)/opt/gnu-tar/libexec/gnubin/tar /usr/local/bin/tar;;
+          x86_64-apple-darwin) NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"; brew install poppler;;
+          *) sudo apt-get -y update ; sudo apt-get -y install libpoppler-glib-dev libpoppler-dev libcairo2-dev libglib2.0-dev
+        esac
+
+    - name: Extract crate information
+      run: |
+        echo "PROJECT_NAME=$(sed -n 's/^name = "\(.*\)"/\1/p' Cargo.toml | head -n1)" >> $GITHUB_ENV
+        # Get the project version from mix.exs
+        echo "PROJECT_VERSION=$(sed -n 's/^  @version "\(.*\)"/\1/p' ../../mix.exs | head -n1)" >> $GITHUB_ENV
+
+    - name: Install Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        target: ${{ matrix.job.target }}
+        override: true
+        profile: minimal
+
+    - name: Show version information (Rust, cargo, GCC)
+      run: |
+        gcc --version || true
+        rustup -V
+        rustup toolchain list
+        rustup default
+        cargo -V
+        rustc -V
+        rustc --print=cfg
+
+    - name: Download cross from GitHub releases - not darwin
+      uses: giantswarm/install-binary-action@v1.0.0
+      if: ${{ matrix.job.use-cross && matrix.job.target != 'aarch64-apple-darwin' }}
+      with:
+        binary: "cross"
+        version: "v0.2.1"
+        download_url: "https://github.com/rust-embedded/cross/releases/download/${version}/cross-${version}-x86_64-unknown-linux-gnu.tar.gz"
+        tarball_binary_path: "${binary}"
+        smoke_test: "${binary} --version"
+
+    - name: Download cross from GitHub releases - darwin
+      uses: giantswarm/install-binary-action@v1.0.0
+      if: ${{ matrix.job.use-cross && matrix.job.target == 'aarch64-apple-darwin' }}
+      with:
+        binary: "cross"
+        version: "v0.2.1"
+        download_url: "https://github.com/rust-embedded/cross/releases/download/${version}/cross-${version}-x86_64-apple-darwin.tar.gz"
+        tarball_binary_path: "${binary}"
+        smoke_test: "${binary} --version"
```

```suggestion
    - name: Download cross from GitHub releases
      uses: giantswarm/install-binary-action@v1.0.0
      if: ${{ matrix.job.use-cross }}
      with:
        binary: "cross"
        version: "v0.2.1"
        download_url: "https://github.com/rust-embedded/cross/releases/download/${version}/cross-${version}-x86_64-unknown-linux-gnu.tar.gz"
        tarball_binary_path: "${binary}"
        smoke_test: "${binary} --version"
```

i think this is not needed anymore since we don't cross compile to the M1